CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ This project adheres to [Semantic Versioning](https://semver.org/).
   (#[1456](https://github.com/nix-rust/nix/pull/1456))
 - Added `TcpUserTimeout` socket option (sockopt) on Linux and Fuchsia.
   (#[1457](https://github.com/nix-rust/nix/pull/1457))
+- Added `renameat2` for Linux(#[1458](https://github.com/nix-rust/nix/pull/1458))
+  (#[1453](https://github.com/nix-rust/nix/pull/1453))
```

Comment:
Somehow you duped a line in the CHANGELOG.
```suggestion
```

Reply:
Gah!  Corrected.  There had been some interleaving of lines with my last merge of master in to my PR branch.  I've squashed it all in to a single commit again.